packages/rum/src/domain/record/startFullSnapshots.ts

```diff
@@ -65,17 +67,31 @@ export function startFullSnapshots(
     return records
   }
 
-  fullSnapshotCallback(takeFullSnapshot())
+  fullSnapshotReadyCallback(takeFullSnapshot())
 
+  let cancelIdleCallback: (() => void) | undefined
   const { unsubscribe } = lifeCycle.subscribe(LifeCycleEventType.VIEW_CREATED, (view) => {
     flushMutations()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is this `flushMutations()` necessary?